opencolorio_config_aces/config/reference/generate/config.py

```diff
@@ -396,128 +496,178 @@ def ctl_transform_to_colorspace(ctl_transform,
+                        ColorspaceDescriptionStyle.SHORT_UNION,
+                        ColorspaceDescriptionStyle.LONG_UNION):
+            aces_transform_ids, aces_descriptions = zip(
+                *[(ctl_transform.aces_transform_id.aces_transform_id,
+                   ctl_transform.description)
+                  for ctl_transform in ctl_transforms])
+
+            if len(description) > 0:
+                description.append('')
+
+            if describe in (ColorspaceDescriptionStyle.ACES
+                            | ColorspaceDescriptionStyle.SHORT,
+                            ColorspaceDescriptionStyle.SHORT_UNION):
+                description.extend([
+                    f'ACEStransformID: {aces_transform_id}'
+                    for aces_transform_id in aces_transform_ids
+                ])
+            else:
+                description.append(
+                    f'CTL Transform'
+                    f'{"s" if len(aces_transform_ids) >= 2 else ""}')
+                description.append(f'{"=" * len(description[-1])}\n')
+
+                description.append(f'\n{"-" * 80}\n\n'.join([
+                    (f'{aces_descriptions[i]}\n\n'
+                     f'ACEStransformID: {aces_transform_id}\n')
+                    for i, aces_transform_id in enumerate(aces_transform_ids)
+                ]))
+
+        description = '\n'.join(description)
+
+    return view_transform_factory(
+        name, from_reference=builtin_transform, description=description)
 
 
-def node_to_colorspace(graph, node, complete_description=True):
+@required('OpenColorIO')
+def style_to_display_colorspace(
+        style, describe=ColorspaceDescriptionStyle.OPENCOLORIO):
     """
-    Generates the *OpenColorIO* colorspace for given *aces-dev* conversion
-    graph node.
+    Creates an *OpenColorIO* display colorspace for given style.
 
     Parameters
     ----------
-    graph : DiGraph
-        *aces-dev* conversion graph.
-    node : unicode
-        Node name to generate the *OpenColorIO* colorspace for.
-    complete_description : bool, optional
-        Whether to use the full *ACES* *CTL* transform description or just the
-        first line.
+    style : unicode
+        *OpenColorIO* builtin transform style
+    describe : int, optional
+        Any value from the
+        :class:`opencolorio_config_aces.ColorspaceDescriptionStyle` enum.
 
     Returns
     -------
     ColorSpace
-        *OpenColorIO* colorspace.
+        *OpenColorIO* display colorspace for given style.
     """
 
-    ctl_transform = node_to_ctl_transform(graph, node)
+    import PyOpenColorIO as ocio
 
-    colorspace = ctl_transform_to_colorspace(
-        ctl_transform,
-        complete_description,
-        to_reference_transform=node_to_builtin_transform(graph, node),
-        from_reference_transform=node_to_builtin_transform(
-            graph, node, 'Reverse'))
+    name = beautify_display_name(style)
+    builtin_transform = ocio.BuiltinTransform(style)
```

Comment:
This would be a discussion for the group, but we may want to add "Display - " in front of these names.

Reply:
Ditto!